playwright/tests/common/sideNav.spec.ts

```diff
@@ -0,0 +1,148 @@
+import { expect } from "@playwright/test";
+
+import { test } from "../../fixtures";
+import { generateLibraries, generateTypes } from "../../mocks";
+
+test.describe("Side nav", () => {
```

Comment:
This isn't a comment specific to this PR: Do you think the `describe` groups are necessary?

We seem to have one `describe` per `.spec.ts` file. The Playwright UI list could be shallower if we removed the `describe` groups.

Reply:
YESSSS! I had the same thought and I want to remove them ALL!

Because we prefer to have small file, and so create another file instead of another describe, we can and should remove them. 
I will do it directly before merging 🙂